webhook/icinga.go

```diff
@@ -137,16 +134,20 @@ func createServiceData(hostname string,
 		// periods are already managed by Prometheus/Alertmanager and relevant
 		// config parameter defaults to 1, but is still tunable for other usecases
 		MaxCheckAttempts: float64(config.MaxCheckAttempts),
+		// Set defaults
+		BridgeUUID: config.UUID,
+		KeepFor:    config.KeepFor,
```

Comment:
Instead of providing fields in the Service object for some application-specific custom variables (these fields make no sense in the public API of the generic Icinga2 client library, as noted in the go-icinga2-client PR), we should manage all variables separately from the service object.